modules/openy_gc_log/openy_gc_log.install

```diff
@@ -186,3 +186,16 @@ function openy_gc_log_update_8010() {
     'views.view.virtual_y_logs',
   ]);
 }
+
+/**
+ * Import updated logs view config.
+ */
+function openy_gc_log_update_8011() {
+  $config_dir = drupal_get_path('module', 'openy_gc_log');
+  $config_dir .= '/config/install/';
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
```

Comment:
@TomScarboroughYMCA why do you want to comment this code?

Reply:
To be honest - I completely cut and pasted the _update blocks I found above this one, which had those comments. I don't have strong feelings on it.

Reply:
@anpolimus can you please clarify your question? The docblock is required as per coding standards, but is there another comment that should go away?

Reply:
@froboy, @TomScarboroughYMCA sorry. I've enabled dark mode at my Github and it was looking like you have commented code of the hook_update.
SOrry.
Everything is fine.